TesseractOCR/build_dependencies.sh

```diff
@@ -7,8 +7,9 @@
 
 GLOBAL_OUTDIR="`pwd`/build"
 LOCAL_OUTDIR="./outdir"
-LEPTON_LIB="`pwd`/leptonica-1.71"
 TESSERACT_LIB="`pwd`/tesseract-3.03"
+LEPTON_LIB="`pwd`/leptonica-1.72"
+IMAGE_LIBS="`pwd`/libtiff-ios-master/dependencies"
```

Comment:
No, please update the README files (both the "how to compile libraries" one and the main root one) as I described *as part of this pull request* because you can just include the edits as an additional commit in this pull request. It helps keep our version history clean, plus the README change is directly related to the changes you made in this pull request, so it just makes sense to include it as part of this pull request.

Git submodules have nothing to do with CocoaPods; they're two completely different things. Git submodules are quite easy to setup and use, so please either read the link I shared explaining how to use them or find a similar article about how to use them via searching Google. If you'd rather do this part as a separate pull request then I'm okay with that, but I think you should make that pull request for the libtiff submodule and we should merge it *first* before we try to merge this pull request to add PDF support. That way the PDF support PR cleanly integrates with the libtiff git submodule PR.

Lastly, the sources we use for upstream Tesseract and Leptonica when compiling the libraries are not Git repositories (they're just zip files that we ask the user to download from the links in the README_howto_compile_libaries.md file) and therefore they cannot exist as Git submodules.